src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -204,6 +204,10 @@ def __str__(self):
 
 
 KNOWN_CLOUDS = [AZURE_PUBLIC_CLOUD, AZURE_CHINA_CLOUD, AZURE_US_GOV_CLOUD, AZURE_GERMAN_CLOUD]
+for kc in KNOWN_CLOUDS:
+    # pylint: disable=line-too-long
+    setattr(kc.endpoints, 'vm_image_alias_doc',
+            'https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/arm-compute/quickstart-templates/aliases.json')
```

Comment:
Can you set it individually for each known cloud?
I prefer all the endpoints to be set on the constructors for each `Cloud` class.
https://github.com/Azure/azure-cli/blob/master/src/azure-cli-core/azure/cli/core/cloud.py#L149